pages/style-guide/prose.md

```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
```

Comment:
@arthurvr  Thanks for the clarification.

You are right: These is my first contribution to this repository so I am not the right guy to handle this. Let's wait to hear other people's opinion.